docs/contrib_components/api/usage.rst

```diff
@@ -0,0 +1,671 @@
+                    "type": "demo.BlogPage"
+                },
+                "title": "My blog 3"
+            }
+        ]
+    }
+
+
+Specifying a list of fields to return
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As you can see, we still only get the ``title`` field, even though we have selected a type. That's because listing pages require you to explicitly tell it what extra fields you would like to see. You can do this with the ``fields`` query parameter.
+
+Just set ``fields`` to a command-separated list of field names that you would like to use.
+
+.. code-block:: json
+
+    GET /api/v1/pages/?type=demo.BlogPage&fields=title,date_posted,feed_image
+
+    HTTP 200 OK
+    Content-Type: application/json
+
+    {
+        "meta": {
+            "total_count": 3
+        },
+        "pages": [
+            {
+                "id": 4,
+                "meta": {
+                    "type": "demo.BlogPage"
+                },
+                "title": "My blog 1",
+                "date_posted": "2015-01-23",
+                "feed_image": 1
+            },
+            {
+                "id": 5,
+                "meta": {
+                    "type": "demo.BlogPage"
+                },
+                "title": "My blog 2",
+                "date_posted": "2015-01-24",
+                "feed_image": 2
+            },
+            {
+                "id": 6,
+                "meta": {
+                    "type": "demo.BlogPage"
+                },
+                "title": "My blog 3",
+                "date_posted": "2015-01-25",
+                "feed_image": 3
+            }
+        ]
+    }
+
+
+We now have enough information to make a basic blog listing with a feed image and date that the blog was posted.
+
+
+Filtering on fields
+^^^^^^^^^^^^^^^^^^^
+
+Exact matches on field values can be done by using a query parameter with the same name as the field. Any pages with the field that exactly matches the value of this parameter will be returned.
+
+.. code-block:: json
+
+    GET /api/v1/pages/?type=demo.BlogPage&date_posted=2015-01-24
+
+    HTTP 200 OK
+    Content-Type: application/json
+
+    {
+        "meta": {
+            "total_count": 1
+        },
+        "pages": [
+
+            {
+                "id": 5,
+                "meta": {
+                    "type": "demo.BlogPage"
+                },
+                "title": "My blog 2",
+                "date_posted": "2015-01-24",
+                "feed_image": 2
+            }
+        ]
+    }
+
+
+Filtering by section of the tree
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is also possible to filter the listing to only include pages with a particular parent. This is useful if you have multiple blogs on your site and only want to view the contents of one of them.
+
+For example (imagine we are in the same project as all previous examples, and page id ``7`` refers to the other blog index):
+
+.. code-block:: json
+
+    GET /api/v1/pages/?type=demo.BlogPage&child_of=7
```

Comment:
`type` is not a required parameter to use `child_of`, so should be removed from the example.